shared/common-adapters/plain-input.desktop.tsx

```diff
@@ -317,6 +318,9 @@ const styles = Styles.styleSheetCreate(() => ({
       outline: 'none',
     },
   }),
+  resize: Styles.platformStyles({
+    isElectron: {resize: 'both'},
```

Comment:
resize both leads to being able to do this - maybe not intended?
<img width="1045" alt="Screen Shot 2019-12-20 at 2 53 04 PM" src="https://user-images.githubusercontent.com/54032873/71288211-7073cf00-2338-11ea-9f0f-1640855888f4.png">

Reply:
Good point. I'll use `vertical` instead.